numpyro/handlers.py

```diff
@@ -241,12 +241,12 @@ def process_message(self, msg):
 
 class condition(Messenger):
     """
-    Conditions unobserved sample sites to values from `param_map` or `condition_fn`.
+    Conditions unobserved sample sites to values from `data` or `condition_fn`.
     Similar to :class:`~numpyro.handlers.substitute` except that it only affects
     `sample` sites and changes the `is_observed` property to `True`.
 
     :param fn: Python callable with NumPyro primitives.
-    :param dict param_map: dictionary of `numpy.ndarray` values keyed by
+    :param dict data: dictionary of `numpy.ndarray` values keyed by
```

Comment:
What do you think about raising a `FutureWarning` with `param_map`? We can remove all of these deprecated arguments in the next release.

Reply:
Sure, I'll add a FutureWarning for this.